src/frontend/src/components/items/CopyButton.tsx

```diff
@@ -0,0 +1,28 @@
+import { t } from '@lingui/macro';
+import { Button, CopyButton as MantineCopyButton } from '@mantine/core';
+import { IconCopy } from '@tabler/icons-react';
+
+export function CopyButton({
+  value,
+  label
+}: {
+  value: any;
+  label?: JSX.Element;
+}) {
+  return (
+    <MantineCopyButton value={value}>
+      {({ copied, copy }) => (
+        <Button
+          color={copied ? 'teal' : 'gray'}
+          onClick={copy}
+          title={t`copy to clipboard`}
```

Comment:
Please capitalise this `Copy to clipboard`

Reply:
done